kodi_addon_checker/check_dependencies.py

```diff
@@ -26,12 +29,17 @@ def check_addon_dependencies(report: Report, repo_addons: dict, parsed_xml, bran
 
     deps = _get_users_dependencies(parsed_xml)
     ignore = _get_ignore_list(branch_name)
-
+    print(deps)
     for required_addon, required_version in deps.items():
         if required_addon not in repo_addons:
             if required_addon not in ignore:
                 report.add(Record(
                     PROBLEM, "Required addon %s not available in current repository." % required_addon))
+            elif required_addon == "xbmc.python":
```

Comment:
As already mentioned on slack, no specific version id hacks. Make this check general, no hard coded addon id  in the `check_addon_dependencies` function.

Reply:
General as in for all dependencies  i.e `xbmc.python`, `xbmc.gui` etc ?

Reply:
One has to be able to add another addon id and version in a list, instead of adding here another elif for further addons.